packages/protocol/migrationsConfig.js

```diff
@@ -95,7 +95,7 @@ const DefaultConfig = {
     participationBaselineQuorumFactor: 1,
   },
   governanceApproverMultiSig: {
-    //Placeholder until addresses for 2/5 multsig are generated.
+    // Placeholder until addresses for 2/5 multsig are generated.
```

Comment:
We have three of these, which I think is enough for now. Can we include them here, and make both required confirmations 2?